pcdscalc/diffraction.py

```diff
@@ -0,0 +1,156 @@
+"""
+Module that handles calculations for diffraction/crystallography.
+
+Assume all the calculations that expect energy as input will
+require energy in eV.
+"""
+import numpy as np
+
+from .constants import lattice_parameters, units, alias
+from .common import cosd, sind, asind
+
+# wavelength/energy = 12398.4 (A) / E(eV)
+WAVELENGTH_TO_ENERGY_LAMBDA = 12398.4
+
+
+def energy_to_wavelength(energy):
+    """
+    Compute photon wavelength in m.
+
+    Parameters
+    ----------
+    energy : number
+        Photon energy in eV.
+
+    Returns
+    -------
+    wavelength : float
+        Wavelength [m].
+    """
+    return (WAVELENGTH_TO_ENERGY_LAMBDA / energy) / units["ang"]
+
+
+def wavelength_to_energy(wavelength):
```

Comment:
Some minor reorganizing: I think this should go in `common` (you'll find it reused everywhere)

Reply:
moved it to `common` in 9880d1d